networkit/cpp/auxiliary/BucketPQ.cpp

```diff
@@ -92,8 +97,9 @@ void BucketPQ::remove(const index& value) {
     }
 }
 
-std::pair<int64_t, index> BucketPQ::extractMin() {
+std::pair<int64_t, index> BucketPQ::extractMin(bool removeElement) {
```

Comment:
This is an awkward signature. If removeElement = true, why doesn't the caller use getMin()?